app/assets/stylesheets/application_redesign.scss

```diff
@@ -132,4 +132,14 @@ summary .bi-chevron-right::before {
 label[for="user-header-hide"] {
   margin-left: -1.5em;
   margin-top: .6em;
+}
+
+input[type="radio"] {
+  width: 22px;
+  height: 22px;
+  margin-right: 4px;
+}
+
+.new_patron_request input:not([type=checkbox]):not([type=radio]):not([type=submit]) {
```

Comment:
What's this styling controlling? It's a little confusing; is there a standard bootstrap way to do this?